React-Calculator/src/specs/utils.test.js

```diff
@@ -0,0 +1,112 @@
+        })
+        test(' addition where both numbers are negative ', ()=>{
+            expect(operations['+'](-3, -3)).toBe(-6)
+        })
+        test(' addition with decimal points on both numbers ', ()=>{
+            expect(operations['+'](0.5, 0.7)).toBe(1.2)
+        })
+        test(' addition where result is expected to have a decimal ', ()=>{
+            expect(operations['+'](2, 0.5)).toBe(2.5)
+        })
+    })
+    describe('possible subtractions', ()=>{
+        test(' subtraction only zeros ', ()=>{
+            expect(operations['-'](0, 0)).toBe(0)
+        })
+        test(' subtraction subtraction where "x" is negative ', ()=>{
+            expect(operations['-'](-2, 9)).toBe(-11)
+        })
+        test(' subtraction subtraction where "y" is negative ', ()=>{
+            expect(operations['-'](5, -8)).toBe(13)
+        })
+        test(' subtraction where both numbers are negative ', ()=>{
+            expect(operations['-'](-3, -3)).toBe(0)
+        })
+        test(' subtraction where both numbers are decimals ', ()=>{
+            expect(operations['-'](0.5, 0.7)).toBe(-0.2)
+        })
+        test(' subtraction where result is expected to contain a decimal point ', ()=>{
+            expect(operations['-'](2, 0.5)).toBe(1.5)
+        })
+    })
+    describe('possible multiplications', ()=>{
+        test(' multiplication only zeros ', ()=>{
+            expect(operations['*'](0, 0)).toBe(0)
+        })
+        test(' multiplication where "x" is negative ', ()=>{
+            expect(operations['*'](-2, 9)).toBe(-18)
+        })
+        test(' multiplication where "y" is negative ', ()=>{
+            expect(operations['*'](5, -8)).toBe(-40)
+        })
+        test(' multiplication where both numbers are negative ', ()=>{
+            expect(operations['*'](-3, -3)).toBe(9)
+        })
+        test(' multiplication with a 1 ', ()=>{
+            expect(operations['*'](2, 1)).toBe(0.5)
+        })
+        test(' multiplication where "y" is a decimal ', ()=>{
+            expect(operations['*'](2, 0.5)).toBe(1)
+        })
+    })
+    describe('possible divisions', ()=>{
+        test(' division only zeros ', ()=>{
+            expect(operations['/'](0, 0)).toBe(0)
+        })
+        test(' division where "x" is negative ', ()=>{
+            expect(operations['/'](-9, 2)).toBe(-4.5)
+        })
+        test(' division where "y" is negative ', ()=>{
+            expect(operations['/'](5, -5)).toBe(-1)
+        })
+        test(' division where both numbers are negative ', ()=>{
+            expect(operations['/'](-3, -3)).toBe(1)
+        })
+        test(' division where both numbers are decimals ', ()=>{
+            expect(operations['/'](0.5, 0.5)).toBe(1)
+        })
+        test(' division where "y" is a decimal ', ()=>{
+            expect(operations['/'](2, 0.5)).toBe(4)
+        })
+        test(' division where divisor is 0 ', ()=>{
+            expect(operations['/'](2, 0)).toBe(Infinity)
```

Comment:
this is _kind of correct_ but, does it work? It could but I never tried it, wanted to double check you did. If you did and it works, then its great! 🚀